data-imports/import-pluto.py

```diff
@@ -229,7 +229,7 @@ def import_csv(args):
 
         sys.stdout.write("\rConcatenating Pluto boro csvs....")
         pluto_csv_files_dir = os.path.join(csv_dir, "BORO_zip_files_csv")
-        pandas_concat_csv(pluto_csv_files_dir, csv_file)
+        simple_concat_csv(pluto_csv_files_dir, csv_file)
```

Comment:
The issue with Pluto, as mentioned to me by Ziggy, is that the schema changes on an almost yearly basis. A quick look over the CSVs didn't reveal any differences but I didn't look super closely.

Part of the reason I was running into these memory issues was I had Docker set to cap out at 2GB; I ran into a similar issue with the web stuff and got around it by lifting the cap to 4GB.

I'll undo the pluto CSV concat change for now. I think the real best solution is to actually import each year individually.